SnapshotTests/Carthage/Checkouts/ios-snapshot-test-case/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD License
```

Comment:
Is it required to have this checked-in in our sources?

Reply:
Do we actually need Checkouts at all in our project? Is it a trash or there is a point with a license for example and so on.